docs/multi_process.md

```diff
@@ -64,7 +64,11 @@ The API {func}`jax.distributed.initialize` takes several arguments, namely:
     cluster will connect.
   * `num_processes`: the number of processes in the cluster
   * `process_id`: the ID number of this process, in the range `[0 ..
-  num_processes)`.
+    num_processes)`.
+  * `local_device_ids`: Restricts the visible devices of the current process to
+    ``local_device_ids``. Defaults to all local devices being visible to the
+    process except when processes are launched via Slurm on GPUs.
+    In that case, it will default to a single device per process.
```

Comment:
Let's actually remove the whole "Defaults to ... single deivce per process." section, since you explain that immediately below. (Sorry if I'm contradicting myself, I just looked at the rendered docs and thought it was a little awkward to read it twice in short succession.)